app/assets/stylesheets/theme/panels.scss

```diff
@@ -124,6 +117,16 @@ $brave-panels-borderRadius: 8px;
     margin-top: auto;
     margin-bottom: auto;
 
+    &--email-sent {
+      padding-top: 125px;
+      margin-top: 30px;
+      @include media-breakpoint-up(md) {
+        background-image: url(asset-path("email-2@1x.png"));
+        background-position: center 20px;
+        background-repeat: no-repeat;
+      }
+    }
+
```

Comment:
These styles were moved here because the flash messages in `emailed_auth_token.html.slim` were pushing the paper plane image under the "Email was sent text".  Now the image moves with the with text, so it will always stay above.